dt1_test.cpp

```diff
@@ -74,7 +74,7 @@ int main(int argc, char* argv[])
             (DT1::Tile::HEIGHT / 2) * (ds1.width + ds1.height),
             sf::Color::Transparent);
 
-    auto drawTile = [&](int32_t id, int x, int y, const std::set<int>& orientations) {
+    std::function<void(int32_t, int, int, const std::set<int>&)> drawTile = [&](int32_t id, int x, int y, const std::set<int>& orientations) {
```

Comment:
I'm not sure why you changed to a `std::function`, but for local lambdas I'd suggest to always use auto.
The compiler will do a better job at optimizing it, and `std::function` can allocate.
It is unneeded unless you want to pass it as a non template function.

Reply:
This file is just a quick hack to verify tiles and show a map, it's not meant to be merged.
I changed it to make it recursive for the "sibling tiles".
Thanks for the info, I'll keep in mind in future